lib/index.js

```diff
@@ -74,6 +74,14 @@ HubSpot.prototype.identify = function(identify) {
 
 HubSpot.prototype.track = function(track) {
   var props = track.properties();
```

Comment:
Hmmm i tried this but this will throw errors if there is no `properties.id` because the way this integration works somehow, if theres no `properties.id` it rightfully sets the `id` as the event name